### DISCUSSION TITLE
Activation function - chapter 3 model: FashionMNISTModelV1

### DISCUSSION THREAD

**Discussion by gkrampah** [mrdbourke/pytorch-deep-learning#1085] [Q&A]
correct me if I am wrong, but shouldn't the model return the raw output without any activation layer since this is implemented in the cross entropy loss function?
